docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -37,7 +37,22 @@ return [
 ];
 ```
 
-The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration.
+## Register a storage adapter
+
+To register additional storage adapters, use either application or module configuration (such as `config/autoload/*.global.php` or `module/Application/config/module.config.php`, respectively), and define the configuration key `storage_adapters`.
```

Comment:
Might be helpful to scrape the documentation of other components with this kind of plugin manager configuration. Would love to align the docs somehow, for example:

https://github.com/laminas/laminas-serializer/blob/2.16.x/docs/book/plugin-manager.md

Reply:
Here is another example: https://docs.laminas.dev/laminas-hydrator/v4/plugin-managers/

But it should be explicitly described, as in this example: https://github.com/laminas/laminas-form/pull/196/files#diff-3a14a07ea6ab87492ab3919a659959973bb061f53b2e3a84395a9e0f4c2dae2b